math/mobius/mobius_test.go

```diff
@@ -0,0 +1,49 @@
+// mobius_test.go
+// description: Returns μ(n)
+// author: Akshay Dubey (https://github.com/itsAkshayDubey)
+// see mobius.go
+
+package mobius
+
+import (
+	"testing"
+)
+
+func getTestsForMu() []struct {
+	n        int
+	expected int
+} {
+	var tests = []struct {
+		n        int
+		expected int
+	}{
+		{-1, 1},
+		{0, 1},
+		{2, -1},
+		{3, -1},
+		{95, 1},
+		{97, -1},
+		{98, 0},
+		{99, 0},
+		{100, 0},
+	}
+	return tests
+}
+
+func TestMu(t *testing.T) {
+
```

Comment:
There is no need for this extra space

Reply:
Fixed [here](https://github.com/TheAlgorithms/Go/pull/532/commits/f7431a5908ad8c370b9fdd94c6290a2df921687d)